src/worldeditor/worldeditor.cpp

```diff
@@ -10,12 +10,6 @@
 |  (C) 2003-2009 Florian Hufsky <florian.hufsky@gmail.com>	|
 +----------------------------------------------------------*/
 
-//TODO:
-//1) Continue testing by creating a fully functional new world
-//3) When saving new world, it didn't save the name correct (or at all)
-
-//Checkin
-
```

Comment:
Do all items on this todo list work now?

Reply:
Yes, I pressed N to create a new world, and it saved with the correct name in the correct place.